### PR TITLE
Create headline

### PR DESCRIPTION
**Pull request by pirc25**
At this first milestones of my portafolio, I did:

- Added index.html
- Added style.css
- Created toolbar
- Created headline

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+  <nav class="toolbar">
+    <p id="toolbar-name">Richard Burneo</p>
+    <i id="hamburguer" class="fa fa-bars" aria-hidden="true"></i>
+  </nav>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] For this section, kindly wrap in round a `header` tag. This is in line with best practices when sectioning your navigation basically 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="css/style.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+  <nav class="toolbar">
+    <p id="toolbar-name">Richard Burneo</p>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] For your logo here, make it an `a` tag to signify a link and not use a `p` tag. 
- This is for three reasons: convention, semantics, and user experience(UX).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+      what
+      you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="social-network">
+      <i class="fa-brands fa-github social"></i>
+      <i class="fab fa-linkedin-in social"></i>
+      <i class="fa-brands fa-angellist social"></i>
+      <i class="fa-brands fa-twitter social"></i>
+      <i class="fa-solid fa-m social"></i>
+    </div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] For this section, kindly use a `ul` tag to group the social links,  an `li` tag per icon, and finally an `a` tag for the icons.
- Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  font-family: 'crete round', sans-serif;
+}
+
+.toolbar {
+  height: 50px;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  color: white;
+  align-items: center;
```
Comment (Shedrack-Sunday) [inline css/style.css:42]: - [x] The margin-top on your logo and hamburger needs a little adjustment to match that of the figma design. Making this change will give your users a better experience with your web app.

## Your view:
image

## Figma:
image

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  position: relative;
+}
+
+h1 {
+  color: #ff6b00;
+  font-size: 56px;
+  align-self: center;
+  justify-content: center;
+  margin: 6px 24px;
+  font-family: 'crete round', sans-serif;
```
Comment (Shedrack-Sunday) [inline css/style.css]: - [ ] Kindly review the fonts provided on the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=5eryAHFhbT9gK0l9-0) to match your current design with that on the figma design. Yours seems a little bit bold. the exact width was provided n the link I attached 👍 

## Your view:
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @pirc25     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯 




Highlights
✅All linters are checked
✅ Descriptive commit messages 
  
Required Changes ♻️

- [x] You did very well in this project 👏 . However, there are a few changes required.
- I noticed a few semantic issues based on use cases of tags, which I commented on
- Also, there are slight adjustments you need to make on the fonts and your project will be all good to go 👍 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by christianbiring1 — APPROVED** [review]
Hi @pirc25 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] All changes were implemented as required by the reviewer :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**